erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -610,7 +610,7 @@ def get_item_details(item, project = None):
 			frappe.throw(_("Default BOM for {0} not found").format(item))
 
 	bom_data = frappe.db.get_value('BOM', res['bom_no'],
-		['project', 'allow_alternative_item', 'transfer_material_against', 'item_name'], as_dict=1)
+		['project', 'allow_alternative_item', 'transfer_material_against', 'item_name','raw_material_cost','operating_cost'], as_dict=1)
```

Comment:
Please fix your formatting before pushing

You can do this in your editor - select any additions you've made and press Ctrl/Cmd + Shift + P, and select "Format Selection"

```suggestion
		['project', 'allow_alternative_item', 'transfer_material_against', 'item_name', 'raw_material_cost', 'operating_cost'], as_dict=1)
```